packages/buidler-core/src/builtin-tasks/compile.ts

```diff
@@ -662,7 +662,7 @@ export default function () {
         quiet,
       }: {
         compilationJob: CompilationJob;
-        emittedArtifactsPerFile: EmittedArtifactsPerFile;
+        emittedArtifactsPerFile: ArtifactsEmittedPerFile;
```

Comment:
There are a bunch fo things like this, where the rename was incomplete. They are in `src/internal/artifacts.ts` and `src/builtin-tasks/compile.ts`.

I was curious, so just run `rg -i emittedartifact`.

Reply:
Ugh, I don't know how I missed that. Fixed, thanks.